.github/workflows/builds.yml

```diff
@@ -171,16 +171,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-11, macos-12]
+       include:
+         - os: macos-11
+           qt-version: 5.15
+           qt-major: 5
```

Comment:
I don't like repeating the variables here. Can we use any of the string manipulation functions available in GH Actions to extract the first '5.' or '6.' in `qt-version`?

That way, we can just use one variable.

Reply:
I plan to improve such things when i unify all the build steps

Reply:
I think I'd be in favour of improving these things at the beginning.